2022q4/FreshPorts.md

```diff
@@ -0,0 +1,59 @@
+## Status Report Sample - This Will Become The Title ##
+
+Link:	 https://freshports.org/[FreshPorts]
+Link:	 https://news.freshports.org/[FreshPorts blog]
+
+Contact: Dan Langille <dan@FreeBSD.org>  
+
+FreshPorts, and its sister site, FreshSource, have reported
+upon FreeBSD commits for 20 years. They cover all commits,
+not just ports.
+
+FreshPorts tracks the commits and extracts data from the
+port Makefiles to create a database of information useful
+to both port developers and port users.
+
+For example, https://www.freshports.org/security/acme.sh/ shows
+the history of this port, back to its creation in May 2017.
+
+### converting the backend repository ###
+
+This topic deals with the FreshPorts code repository. The
+front end (website) was converted from subversion to git several
+years ago. The back end, which processes FreeBSD commits and
+updates the database, is still on subversion. I have
+wanted to convert these repositories to git for some time.
+
+I would like help with this please. I'll give you a copy of the 
+repositories and you give me back a git repo. It will be uploaded
+to https://github.com/FreshPorts
```

Comment:
Missing period.

Reply:
I actively try avoid a period after a URL for fear of 404s.

Reply:
How about `https://github.com/FreshPorts[https://github.com/FreshPorts].`?

Reply:
(Since there are other bare URLs and it's a mechanical change that doesn't change content, I can take care of them after or while merging.)